book/src/csi-windows.md

```diff
@@ -0,0 +1,93 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+Alpha | 1.18 | -
+Beta | 1.19 | -
+
+## Overview
+
+CSI drivers (e.g. AzureDisk, GCE PD, etc.) are recommended to be deployed as containers. CSI driver’s node plugin typically runs on every worker node in the cluster (as a DaemonSet). Node plugin containers need to run with elevated privileges to perform storage related operations. However, Windows currently does not support privileged containers. To solve this problem, [CSI Proxy](https://github.com/kubernetes-csi/csi-proxy) - that runs on the Windows host and executes a set of privileged storage operations on Windows nodes on behalf of containers in a CSI Node plugin daemonset. This enables multiple CSI Node plugin authors to execute privileges storage operations on Windows nodes without having to ship a custom privileged operation proxy (until privileged operations get enabled for containers in Windows in the future).
+
+## How to use the CSI Proxy for Windows?
+The CSI Proxy executable - csi-proxy.exe - needs to be deployed and running on all Windows nodes in a cluster - similar to kubelet.exe, dockerd.exe, etc. This may be achieved by enhancing any existing Windows node setup mechanism or script used for installing and configuring dockerd.exe/containerd.exe, kubelet.exe, kubeproxy.exe, etc.
```

Comment:
The instructions for csi-proxy deployment should go in the csi-proxy page, and you can link to it from here

Reply:
done